.github/workflows/pr_test_peft_backend.yml

```diff
@@ -110,23 +110,23 @@ jobs:
         python -m venv /opt/venv && export PATH="/opt/venv/bin:$PATH"
         python -m pytest -n 4 --max-worker-restart=0 --dist=loadfile \
           -s -v \
-          --make-reports=tests_${{ matrix.config.report }} \
+          --make-reports=tests_${{ matrix.lib-versions }} \
```

Comment:
This change:

* Removes the faulty `matrix.config.report` because there's nothing called config in the matrix. 
* Adds `lib-version` (which is in the `matrix`) to make the names unique.